site/docs/v1/tech/5-minute-setup-guide.adoc

```diff
@@ -90,7 +90,7 @@ fusionauth/bin/startup.sh
 
 This will start both the `fusionauth-app` component as well as the `fusionauth-search` component if you downloaded the Elasticsearch option. (Here's link:/docs/v1/tech/core-concepts/users#user-search[a document on how to choose whether to do so].) 
 
-If you are on Windows, you'll need to use the `startup.bat` file instead.
+If you are on Windows, you'll need to use the `startup.ps1` file instead.
```

Comment:
If we decide to keep two versions of the page, do we need to create a copy of this one?